contrib/destination-starrocks/src/main/java/io/airbyte/integrations/destination/starrocks/StarRocksConstants.java

```diff
@@ -65,13 +65,15 @@ public interface StarRocksConstants {
 
     //csv
     interface CsvFormat {
-        String LINE_DELIMITER = "\n";
-        byte[] LINE_DELIMITER_BYTE = LINE_DELIMITER.getBytes(StandardCharsets.UTF_8);
+        String LINE_DELIMITER = "\\x02";
```

Comment:
Why not change `LINE_DELIMITER` from "\n" to "\\x02"?

Reply:
Change LINE_DELIMITER from "\n" to "\x01" because String/text may contains character “\n”